docker-compose-mysql.yaml

```diff
@@ -54,13 +52,7 @@ services:
     - /opt/mailman/database:/var/lib/mysql
     networks:
       mailman:
-        ipv4_address: 172.19.199.4
 
 networks:
    mailman:
      driver: bridge
-     ipam:
```

Comment:
We can get rid of the entire network configuration tbh, I don't know if there is really a need to configure it at all.

Compose will create a default bridge and add all containers, so there is no need to explicitly specify that.

Reply:
Yes, it will work with the default bridge. But it is good practice to use a user-defined bridged:
https://docs.docker.com/network/bridge.

Reply:
Ah okay, that's something new for me. Thanks!